cg/meta/report/templates/macros/data_analysis/qc_metrics/rnafusion_qc_metrics.html

```diff
@@ -3,7 +3,8 @@
 {% macro rnafusion_qc_metrics(samples) %}
   {% set
     metrics = [
-      {"name": "RNA Integrity Number (RIN)",                   "key": "rin",                   "description": "Mått på skala från 1 till 10 för kvalitet och integritet hos RNA-prover."},
+      {"name": "RNA Integrity Number (RIN)",                   "key": "rin",                   "description": "Mått på skala från 1 till 10 för kvalitet och integritet hos RNA-prover. N/A som ett värde är en varning att RIN kanske inte är tillförlitlig för det specifika provet."},
+      {"name": "DV200",                                        "key": "dv200",                 "description": "Andelen RNA-fragment längre än 200 nukleotider. Denna metrik fungerar som en kvalitetsindex och kategoriserar prover som hög kvalitet (> 70%), medelkvalitet (50–70%), låg kvalitet (30–50%) eller för degraderade (< 30%)."},
```

Comment:
Nice! out of curiosity, where did you get this from?

Reply:
Was about to write you a slack regarding this to confirm the values.

But it's from [illumina's guidelines](https://www.illumina.com/content/dam/illumina-marketing/documents/products/technotes/evaluating-rna-quality-from-ffpe-samples-technical-note-470-2014-001.pdf). The only thing is that the study is only about FFPE samples.

Reply:
Let's double check with the lab if they interpret these values in the same way. Otherwise we can skip the second line.

Reply:
the DV200, do we obtain this from the sequencing? Just curious

Reply:
no, it's calculated during the RNA reception QC step with the Tapestation

Reply:
> Let's double check with the lab if they interpret these values in the same way. Otherwise we can skip the second line.

[For now we are only recording the DV200-values and they don't need to be evaluated.](https://atlas.scilifelab.se/production/lab/supporting_procedures/rc_rna/#322-tapestation).

Should we keep all of those thresholds or just keep the bottom one (<30%) in the description? @fevac

Reply:
I would say we only keep: `"description": "Andelen RNA-fragment längre än 200 nukleotider."`